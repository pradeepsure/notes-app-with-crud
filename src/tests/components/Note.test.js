import Note from '../../components/Note';
import React from 'react';
import { shallow, mount } from 'enzyme';
import { createSerializer } from 'enzyme-to-json';

expect.addSnapshotSerializer(createSerializer({mode: 'deep'}));

let wrapper, note, removeNote;
beforeEach(() => {    
    note= {
        "noteTitle": "Amazon slae",
        "noteDescription": "Amazon Kitchen items sale upto 60% off",
        "id": 1
        }
      removeNote = jest.fn();
    wrapper = shallow(
        <Note note={note} removeNote={removeNote} />
    );
});


test('Render Note component', () => {        
    expect(wrapper).toMatchSnapshot();
});