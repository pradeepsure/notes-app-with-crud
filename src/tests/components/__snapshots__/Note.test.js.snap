// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Note component 1`] = `
<Note
  classes={
    Object {
      "deleteIcon": "Note-deleteIcon-1",
    }
  }
  note={
    Object {
      "id": 1,
      "noteDescription": "Amazon Kitchen items sale upto 60% off",
      "noteTitle": "Amazon slae",
    }
  }
  removeNote={[MockFunction]}
/>
`;
